docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  app:
+    image: node:8
+    volumes:
+      - user_data:/root/.skycli
+      - .:/usr/src/app
```

Comment:
can add /usr/src/app/node_modules here

Reply:
only works if you also add a volume further down in the same compose file i.e. `- node_modules:/usr/src/app/node_modules` and `node_modules: driver: local`